ethereum/spec/src/test/java/tech/pegasys/teku/spec/logic/common/helpers/MiscHelpersTest.java

```diff
@@ -179,6 +185,17 @@ public void computesTimeAtSlot(final UInt64 slot, final long expectedTime) {
     assertThat(actualTime).isEqualTo(UInt64.valueOf(expectedTime));
   }
 
+  @ParameterizedTest
+  @MethodSource("getCommitteeComputationArguments")
+  public void committeeComputationShouldNotOverflow(int validatorsCount, int validatorIndex) {
```

Comment:
It's `committeeIndex` rather than `validatorIndex`, So we have all validators, shuffle this list and split into committees

Reply:
My bad, renamed it